launch/no_roof_small_warehouse.launch

```diff
@@ -0,0 +1,16 @@
+<launch>
+  <!-- Always set GUI to false for AWS RoboMaker Simulation
+       Use gui:=true on roslaunch command-line to run with a gui.
+  -->
+  <arg name="gui" default="false"/>
+
+  <include file="$(find gazebo_ros)/launch/empty_world.launch">
+    <arg name="world_name" value="$(find aws_robomaker_small_warehouse_world)/worlds/no_roof_small_warehouse.world"/>
```

Comment:
nit; i'd prefer to pass the world name as an argument and default it. Fewer launch files. If you want a separate launch file, then have this "no roof" launch that simply includes the base .launch file with the argument. Less duplicated logic. Obviously for two files it's not an issue but it's a pattern that will get stamped out over time.